components/match2/wikis/rainbowsix/match_group_input_custom.lua

```diff
@@ -317,10 +319,17 @@ function mapFunctions.getExtraData(map)
 		t1firstside = {rt = map.t1firstside, ot = map.t1firstsideot},
 		t1halfs = {atk = map.t1atk, def = map.t1def, otatk = map.t1otatk, otdef = map.t1otdef},
 		t2halfs = {atk = map.t2atk, def = map.t2def, otatk = map.t2otatk, otdef = map.t2otdef},
-		t1bans = {map.t1ban1, map.t1ban2},
-		t2bans = {map.t2ban1, map.t2ban2},
+		t1bans = {},
+		t2bans = {},
 		pick = map.pick
 	}
+
+	for opponentIndex = 1, MAX_NUM_OPPONENTS do
+		for banIndex = 1, MAX_NUM_BANS do
+			local ban = map['t' .. opponentIndex .. 'ban' .. banIndex]
+			table.insert(map.extradata['t' .. opponentIndex .. 'bans'], CharacterNames[ban and ban:lower()])
```

Comment:
![image](https://github.com/Liquipedia/Lua-Modules/assets/43279191/05f25b4c-3b07-433c-a000-d6f22502dd25)
Is this intended? (mapping none and stuff to recruit)

Reply:
Yes. Recruit is not a valid operator for use in pro games. Any Recruit listed on a pro match was an easy find for an incorrect input. Now they did just revamp recruit so i do not currently know if that will remain the case.